docs/static/jvm.asciidoc

```diff
@@ -0,0 +1,49 @@
+[float]
+[[ls-jvm]]
+=== Java (JVM) version
+
+{ls} requires one of these versions:
+
+* Java 8
+* Java 11
+* Java 14
+
+Use the
+http://www.oracle.com/technetwork/java/javase/downloads/index.html[official
+Oracle distribution] or an open-source distribution, such as
+http://openjdk.java.net/[OpenJDK].
+
+[float]
+[[check-jvm]]
+==== Check your Java version
+Run the following command:
+
+[source,shell]
+java -version
+
+On systems with Java installed, this command produces output similar to the following:
+
+[source,shell]
+-----
+java version "11.0.1" 2018-10-16 LTS
+Java(TM) SE Runtime Environment 18.9 (build 11.0.1+13-LTS)
+Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
+-----
+
+[float]
+[[java-home]]
+==== `JAVA_HOME`
+
+{ls} uses the Java version set in `JAVA_HOME`.
+
+On some Linux systems, you may need to have the `JAVA_HOME` environment
+exported before installing {ls}, particularly if you installed Java from
+a tarball. 
+{ls} uses Java during installation to automatically detect your environment and
+install the correct startup method (SysV init scripts, Upstart, or systemd). If
+{ls} is unable to find the `JAVA_HOME` environment variable during package
+installation, you may get an error message, and {ls} will not start properly.
+
+For information on settting `JAVA_HOME`, see the
+https://docs.oracle.com/cd/E19182-01/820-7851/inst_cli_jdk_javahome_t/[Oracle
```

Comment:
I'm a little wary of linking into Oracle's docs, as they can go stale or disappear. 
(That page has broken image links and "Homepage is misspelt...)

Reply:
As a best practice, we try to link to the owner's docs rather than re-documenting their solution.  You make a valid point in this case. That's a precarious-looking link that we might not want to count on.  I'll come up with something else.